src/org/labkey/test/tests/UserEditorWithoutDeleteTest.java

```diff
@@ -57,19 +53,27 @@ private void doSetup()
     public void testFileDeleteWithEditorWithoutDeleteUser()
     {
         goToProjectHome();
+        goToModule("FileContent");
         impersonate(EDITOR_WITHOUT_DELETE.getEmail());
-        FilesWebPart filesWebPart = FilesWebPart.getWebPart(getDriver());
-        filesWebPart.fileBrowser().uploadFile(TEST_FILE);
+        _fileBrowserHelper.uploadFile(TEST_FILE);
         Assert.assertFalse("Delete action should not be present for " + EDITOR_WITHOUT_DELETE.getEmail(),
-                filesWebPart.fileBrowser().isActionPresent(FileBrowserHelper.BrowserAction.DELETE));
+                _fileBrowserHelper.isActionPresent(FileBrowserHelper.BrowserAction.DELETE));
 
         WebDavUploadHelper uploadHelper = new WebDavUploadHelper(WebDavUrlFactory.webDavUrlFactory(getProjectName()),
                 WebDavUtils.beginSardine(getCurrentUser()));
         uploadHelper.uploadFile(TEST_FILE);
-        uploadHelper.deleteFile(WebDavUtils.buildBaseWebDavUrl(getProjectName()) + TEST_FILE.getName(), true);
+        SardineException sardineException = uploadHelper.deleteExpectingError(WebDavUtils.buildBaseWebDavUrl(getProjectName()) + TEST_FILE.getName());
+        Assert.assertEquals("Incorrect response for deletion", 404, sardineException.getStatusCode());
```

Comment:
The correct status code is 403. You're getting a 404 because you're passing a full URL to `deleteExpectingError` instead of just the file name.